### PR TITLE
528BZ1698198 remove daemonset label and add playbook name and reference

### PR DESCRIPTION
**Pull request by jhoyt-rh**
This PR is for [BZ1698198](https://bugzilla.redhat.com/show_bug.cgi?id=1698198). There were previous PRs related to this bug: 

- PR14812 (https://github.com/openshift/openshift-docs/pull/14812) 
- PR14809 (https://github.com/openshift/openshift-docs/pull/14809)
- PR14617 (https://github.com/openshift/openshift-docs/pull/14617).

Above were closed, unmerged, for several reasons:

- Additional material (playbook name and xref) needed after peer review.
- QE was passing the review around to several people to verify
- I thought it best to consolidate this information for ease of review and final approval from final QE engineer (@knarra@redhat.com), plus the length of time this Doc fix has been in the works due to above reasons. 

@knarra@redhat.com, will you PTAL?

### CODE REVIEW

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
 .. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR`. The
 default value is `storagenode=glusterfs`.
 
-.. Remove the daemonset label from the node:
+.. Add a label (for example, `type=upgrade`) to the node you want to upgrade.
 +
+[NOTE]
+====
+Use the *_upgrade_nodes.yaml_* playbook:
```
Comment (bergerhoffer) [inline upgrading/automated_upgrades.adoc]: I haven't worked enough with the 3.x docs to know if the guidance differs there, but I thought file names just went in backticks (instead of bold and italic).

File: upgrading/automated_upgrades.adoc [hunk 2]
```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
 .. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR`. The
 default value is `storagenode=glusterfs`.
 
-.. Remove the daemonset label from the node:
+.. Add a label (for example, `type=upgrade`) to the node you want to upgrade.
 +
+[NOTE]
```
Comment (bergerhoffer) [inline upgrading/automated_upgrades.adoc]: Just wondering if this has to be in a note? If everyone coming through these steps would use the upgrade_nodes.yaml playbook, I'd assume it'd just be part of this step. Or if it is necessary as a note, maybe reword the intro sentence of the note ("Use the upgrade_nodes.yaml playbook") to be clearer why it's a note. Is it only in certain situations? Is this just an example?

File: upgrading/automated_upgrades.adoc [hunk 3]
```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
...
+====
+Use the *_upgrade_nodes.yaml_* playbook:
 ----
-$ oc label node <node_name> <daemonset_label>-
+# ansible-playbook -i </path/to/inventory/file> \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/upgrades/v3_9/upgrade_nodes.yml \
```
Comment (bergerhoffer) [inline upgrading/automated_upgrades.adoc]: The sentence above calls it upgrade_nodes.yaml, but it's upgrade_nodes.yml here. Should one of these be updated so that they match?

File: upgrading/automated_upgrades.adoc [hunk 4]
```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
...
+    [-e <customized_node_upgrade_variables>] <1>
```
Comment (bergerhoffer) [inline upgrading/automated_upgrades.adoc]: I'm not sure how this command is labeling anything. Unless it's done by this -e flag? If so, maybe update the callout to give an example of how a label might be added here?

File: upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: Good question, but let me tell you why I did it the way I did... I copied the format from previous references in the file. I hope that is ok. I thought it best to use the same format that was a couple of hundred lines above it.

I wonder if it was some time after 3.9 that the format for references changed?

File: upgrading/automated_upgrades.adoc (hunk 3 above)
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: yes, thank you. I'll fix.

File: upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (kalexand-rh) [reply upgrading/automated_upgrades.adoc]: The format for file names changed in the 4.x stream, and these are right for 3.x.

**Comment by vikram-redhat** [comment]
@jhoyt-rh - what are the numbers 528 in the front of title of this PR?

**Comment by jhoyt-rh** [comment]
@vikram-redhat -- the numbers are simply the date. This bug has gone through so many iterations (note the 3 previous PRs listed in the initial comment) that I wanted an easy way to find the correct one to use for this version. So it is for the branch for BZ1698198 on May 28 (528); hence, branch 528BZ1698198.

File: upgrading/automated_upgrades.adoc (hunk 2 above)
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: Good point. I took it out of a note. Step is now using that playbook to run the listed command. The note identifying the location of the playbook now follows that step.

**Comment by jhoyt-rh** [comment]
@ahoffer@redhat.com, PTAL at new edits

@sselvan@redhat.com, PTAL

Thank you.

**Comment by jhoyt-rh** [comment]
@bergerhoffer, will you PTAL at new edits. Thank you.

File: upgrading/automated_upgrades.adoc (hunk 4 above)
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: Thank you, Andrea. Yes, I will ask the QE engineer.

File: upgrading/automated_upgrades.adoc [hunk 5]
```diff
@@ -815,7 +813,11 @@ Ensure all of the volumes are healed and there are no outstanding tasks. The
 A volume is considered healed when `Number of entries` for that volume is `0`.
 
 .. Remove the upgrade label (for example, `type=upgrade`) and go to the next
-GlusterFS node.
+GlusterFS node:
++
+----
+$ oc label node <node_name> type-upgrade
```
[the remark this answers is not in the document]
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: @bergerhoffer Thanks. Updated the syntax with information from SME.

**Comment by jhoyt-rh** [comment]
@bergerhoffer As noted above, I have updated the syntax from SME jrivera@redhat.com.

Will  you please be able to merge this PR for me? Thank you.

**Comment by bergerhoffer** [comment]
Looks good to me, merging!
